config/default/tox.ini.j2

```diff
@@ -313,3 +313,12 @@ commands =
 #  _your own configuration lines_
 #  """
 ##
+
+[testenv:release]
+description = create a new release
+skip_install = true
+deps =
+    zest.release[recommended]
```

Comment:
Typo in `zest.releaser`.  And we should add two that are not in the recommended list:

```suggestion
    zest.releaser[recommended]
    zest.pocompile
    zestreleaser.towncrier
```